astacus/coordinator/plugins/cassandra/plugin.py

```diff
@@ -49,6 +49,7 @@ async def run_step(self, cluster: Cluster, context: StepsContext) -> None:
 class CassandraPlugin(CoordinatorPlugin):
     client: CassandraClientConfiguration
     nodes: Optional[List[CassandraConfigurationNode]]
+    datacenter: Optional[str]
```

Comment:
Sorry, I didn't see that earlier.
If astacus is updated and the config is not updated yet, will astacus fail to start because the value has no default value ?

(I'm thinking about intentional updates but also just a server or package manager pulling the last version by default)

Reply:
according to [pydantic docs](https://docs.pydantic.dev/usage/models/#required-optional-fields), specifying Optional allows to omit the field, and `CoordinatorPlugin` is an `AstacusModel`. I don't think it's necessary to make this a "required optional" field (require saying `datacenter=None` in every configuration), this is an opt-in feature.

Reply:
I agree with you, it should not be required optional :+1: 

I disagree with the choice of syntax of pydantic though :disappointed: